tests/VolleyManagement.Specs/PlayersContext/CreatePlayerSteps.cs

```diff
@@ -59,18 +79,25 @@ public void GivenYearOfBirthIs(short birthYear)
         [Given(@"player without name")]
         public void GivenPlayerWithoutName()
         {
-            _player.FirstName = null;
-            _player.LastName = null;
+            _playerFirstName = null;
+            _playerLastName = null;
         }
 
         [When(@"I execute CreatePlayer")]
         public void WhenIExecuteCreatePlayer()
         {
-            var playerToAdd = Mapper.Map<CreatePlayerDto>(_player);
-
-            _player = _playerService.Create(playerToAdd);
+            try
+            {
+                _player.FirstName = _playerFirstName;
```

Comment:
Well, again. Exception is thrown here when FirstName is invalid. Make it work when try block covers only service call.